extensions/skins/conversation_v1/Conversation.js

```diff
@@ -208,11 +208,22 @@ oppia.directive('conversationSkin', [function() {
       $scope.PANEL_TUTOR = 'tutor';
       $scope.PANEL_SUPPLEMENTAL = 'supplemental';
 
+      $scope.helpHtml = null;
+      $scope.continueToNextCardHtml = null;
+
       $scope.profilePicture = '/images/avatar/user_blue_72px.png';
       oppiaPlayerService.getUserProfileImage().then(function(result) {
         $scope.profilePicture = result;
       });
 
+      $scope.clearHelpHtml = function() {
+        $scope.helpHtml = null;
+      };
+
+      $scope.clearContinueToNextCardHtml = function() {
+        $scope.continueToNextCardHtml = null;
+      };
+
```

Comment:
Also, why have this function at all? Why not just have $scope.helpHtml = null in the code?